zzz_generated/app-common/app-common/zap-generated/cluster-objects.cpp

```diff
@@ -35101,5 +35101,584 @@ bool CommandIsFabricScoped(ClusterId aCluster, CommandId aCommand)
     return false;
 }
 
+bool CommandHasLargePayload(ClusterId aCluster, CommandId aCommand)
+{
+    switch (aCluster)
+    {
+    case Clusters::Identify::Id: {
+        switch (aCommand)
+        {
+        default:
+            return false;
```

Comment:
This is huge. How about we just switch for the "true" set and that is it?

Reply:
The problem is that ZAP does not give us a clean way to enumerate "the true set", right?  @brdandu

Reply:
Yes. That is what I gathered as well. We have to go through all the clusters to check their list of commands and then check the specific quality.

Reply:
You can loop over everything and have a `if true` I imagine. It just needs compressing the switch into a compare.

Reply:
I am generally wondering if this could read:

```cpp
bool CommandHasLargePayload(ClusterId aCluster, CommandId aCommand)
{
  if ((aClusterId == Clusters::CameraAvStreamManagement::Id) 
      && (aCommandId == Clusters::CameraAvStreamManagement::Commands::CaptureSnapshot::Id)
  {
    return true;
  }
  
   if ((aClusterId == Clusters::CameraAvStreamManagement::Id) 
       && (aCommandId == Clusters::CameraAvStreamManagement::Commands::CaptureSnapshotResponse::Id)
  {
    return true;
  }
  
  return false;
}
```

We keep accepting sub-part code because it is code generated. I think we should not do that.

Reply:
Yes, you could certainly generate that code.

Reply:
Yes. I was not too conversant with the zap templates and their code generation model. Was looking at only the switch based templates. But the template for generating the above code seems pretty straight-forward. Thanks.